src/tools/ci/install_meson_and_rebase.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/sh
+
+set -e
+set -x
+
+MESON_REPO=$1
+MESON_BRANCH=$2
+CURRENT_OS=$3
+
+if [ $# -ne 3 ]; then
+    echo "install_meson_and_rebase.sh <MESON_REPO> <MESON_BRANCH> <OS>"
+    exit 1
+fi
+
+case $CURRENT_OS in
+    freebsd|linux|macos|windows|mingw)
+    ;;
+    *)
+        echo "unsupported operating system ${CURRENT_OS}"
+        exit 1
+    ;;
+esac
+
+# After repartition, hidden files are not copied. Copy them to
+# working dir
+if [ "$CURRENT_OS" = 'freebsd' ]; then
+    cp -r $CIRRUS_WORKING_DIR.orig/.[^.]* $CIRRUS_WORKING_DIR/
+    chown -R postgres:postgres .[^.]*
+fi
+
+# pip is not installed as a default on some OSes. First install
```

Comment:
How about we instead change the image definitions so that pip is included? That's also useful for patches to postgres that need other dependencies / utilities.